### ISSUE TITLE
Use SEP from delayed OTA

### ISSUE THREAD

**Issue by blahdy** [marijuanARM/futurerestore#40]
is it possible to implement this while 14.8 is out there?

**Comment by Cryptiiiic** [comment]
Not for downgrading no only upgrading devices that have iOS 14 sep.

**Comment by Cryptiiiic** [comment]
And even then no one knows it needs research

**Comment by Cryptiiiic** [comment]
@blahdy not really the place to even ask this join the discord linked in the README

**Comment by ghost** [comment]
Looks like there’s a lot of people in the sub with an X stuck on < iOS 13.x this might be big if true👀

**Comment by BerlinSnoop** [comment]
Is there something (progress) planned for this? 
This could help for the new not yet finished IOS15-Jailbreak even when the 15.3.1-Window is closed.

**Comment by Cryptiiiic** [comment]
Not really no, this most likely requires you being below 14.8.1 or below the delayed version's os and not downgrading it needs to be on that version when it was being signed because you can't downgrade sep. Even then probably need sep exploit.

**Comment by BerlinSnoop** [comment]
Thanks for the quick response. But Yes, many people will sit with their jailbreak at 14.8 (below 14.8.1) and below 15.3.1 when the signing windows is closed in the next days. I have used the the OTAdelay to jump to 14.8 (from 14.3) and at this moment the normal 14.8 signing window was already closed. Maybe I misunderstood you?

**Comment by Cryptiiiic** [comment]
Delayed OTA/Alt Update uses custom shsh blobs introduced in 14.5 so only 14.5 and later can use them anyway. Unless someone made a tweak I don't remember. Those blobs use something called a BCERT which is generated by the sep itself. Meaning you need a jailbroken device to obtain this cert in the first place. Not only that but the cert expires after 3 days. After those 3 days your delayed ota/alt update blobs will be rendered useless afaik. So this is not really worth the effort of reverse engineering the ota update daemons, drivers, and other software especially when the highly obfuscated and secure sep is involved. It takes too much take and effort for something that only will last 3 days.

**Comment by BerlinSnoop** [comment]
Ok. That clarified it for me. That's a pity. Many thanks for the explanation

**Comment by ghost** [comment]
If you're on, say, 14.3 with 15.3 SEP, it *might* be possible to use a delay OTA SEP blob from 15.3 or above. I will try to look into that. These blobs would be single-use at best as the SEP nonce needs to match (we can't set it), but technically as long the delay OTA doesn't expire and you don't restore your device with a higher SEP you could generate unlimited BCerts, so the 3-day expiry would not be an issue in practice.

In a way this would be similar to nonce freeze, it would have limited usefulness. I would not expect a solution to get delay OTA blobs from a version that's *below* the SEP version you last restored with anytime soon, especially not on A11+ which doesn't have a public SEP exploit.

TL;DR: Maybe it can be used for upgrades in some limited cases, but I definitely don't see it working for downgrades anytime soon.

**Comment by Cryptiiiic** [comment]
Future potential aspiration
